brunton_lab_to_nwb/brunton_lab_to_nwb.py

```diff
@@ -48,7 +48,7 @@ def run_conversion(
     file_elec_col_names = file['chan_info']['axis1'][:]
     elec_data = file['chan_info']['block0_values']
 
-    re_exp = re.compile("([a-zA-Z]+)([0-9]+)")
+    re_exp = re.compile("([ a-zA-Z]+)([0-9]+)")
```

Comment:
Since spaces are included in some electrode group names, the regular expression needs to consider them with the alphabetical component.